src/renderer/render-to-photon.js

```diff
@@ -128,11 +128,15 @@ const {CanvasToImage} = require("./canvas_to_img");
 
         const cornerToCoords = {
             "TL": [x_offset_px, y_offset_px],
-            "TR": [longRes - x_offset_px - board_width_px, y_offset_px],
-            "BL": [x_offset_px, shortRes - y_offset_px - board_height_px],
+            "TR": [outputWidth - x_offset_px - board_width_px, y_offset_px],
+            "BL": [x_offset_px, outputHeight - y_offset_px - board_height_px],
             "BR": [
-                longRes - x_offset_px - board_width_px,
-                shortRes - y_offset_px - board_height_px
+                outputWidth - x_offset_px - board_width_px,
+                outputHeight - y_offset_px - board_height_px
+            ],
+            "C": [
+                (outputWidth / 2) - x_offset_px - (board_width_px / 2),
+                (outputHeight / 2) - y_offset_px - (board_height_px / 2)
```

Comment:
```suggestion
                (outputWidth / 2) + x_offset_px - (board_width_px / 2),
                (outputHeight / 2) + y_offset_px - (board_height_px / 2)
```

Reply:
Hi @Andrew-Dickinson, sorry the late answer.

I have performed the changes you have requested, let me know if anything else is needed.